public/scripts/week-calendar.js

```diff
@@ -850,12 +898,20 @@ function checkMeetingValidations(form) {
     meeting_tag_error.innerText = "tag cant be longer than 20 characters";
     form.tag.setCustomValidity("error");
   }
-  if (!form.tag.value.match(/^[a-zA-Z]+$/)) {
+  if (
+    form.tag.value.trim().length > 0 &&
+    !form.tag.value.match(/^[a-zA-Z]+$/)
+  ) {
     meeting_tag_error.innerText = "tag has only letters with no spaces";
     form.tag.setCustomValidity("error");
   }
 
-  if (form.title.value.length > 100) {
+  if (form.title.value.length < 1) {
```

Comment:
lets remove the pattern validation for title  pattern="^(?=.{1,100}$)(?![\d])[\w\s]+$"
and /^(?![\d])[\w\s]+$/gi

since we are only concerned with length